packages/react/src/components/header/components/headerNavigationMenu/HeaderNavigationMenu.module.scss

```diff
@@ -0,0 +1,36 @@
+.headerNavigationMenu {
+  display: flex;
+  width: 100%;
+  height: 64px;
+  background-color: var(--color-white);
+  border-bottom: 1px solid var(--color-black-20);
+}
+.headerNavigationMenuList {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  gap: 0 var(--spacing-xs);
+  flex-wrap: wrap;
+}
+.headerNavigationMenuLinkContainer {
+  margin-left: 10px;
```

Comment:
Is this exactly 10px or could a --spacing variable be used?

Reply:
It's exactly 10px, I got it from hel.fi navigation. I couldn't seem to get the values from Abstract for some reason. @hannu-kivimaki , I styled according to hel.fi implementation here and there, is that okay?

Reply:
We can use our --spacing-xs, which is 12px. Just to keep things in our system streamlined with no exceptions.

Reply:
I'm not sure why hel.fi has used a tighter spacing there. 🤔